src/transaction.ts

```diff
@@ -356,4 +342,68 @@ export default class Transaction {
     // Note: This never gets executed, defineProperties overwrites it.
     return rlp.encode(this.raw)
   }
+
+  private _validateV(v?: Buffer): void {
+    if (v === undefined || v.length === 0) {
+      return
+    }
```

Comment:
This leads to that ``v`` can be set to ``undefined`` on post Spurious Dragon on the setter, is this intended behavior?

Reply:
Yes, that's correct. It's here for backward compatibility. All of the `defineProperties`-based setters accept `undefined`, which sets its default value (i.e. an empty buffer).

I think that allowing that in every field except `v` would be confusing.

Reply:
Ah, that makes sense, thanks.